jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/BadgeTests.java

```diff
@@ -37,17 +39,15 @@ public void simpleBadgeTest() {
 
     @Test
     public void customBadgeTest() {
-        unlockAccountButton.click();
-        lockAccountButton.click();
-        customBadges.stream().map(Badge::is).forEach(e -> {
-            e.is().displayed();
-            e.has().numberOfNewMessages(-1);
-        });
+        lockUnlockAccountButton.click();
+        customBadges.find(By.tagName("i")).is().classValue(Matchers.containsString("mdi-lock-open-variant"));
```

Comment:
In autotest we usually check something meaningfull: counts, images and so on. 
Checking the class names means nothing. What are you going to test there?

Reply:
The Icon is displayed based on the class attribute, so I check the icon image changing by verifying the class attribute.